src/validation/aero_model/modal_aero_bcscavcoef_init.cpp

```diff
@@ -0,0 +1,50 @@
+// mam4xx: Copyright (c) 2022,
```

Comment:
My comments for `modal_aero_bcscavcoef_get.cpp` also apply to this file, but they're minor, so don't give them too much thought. :-)